service/mailService.js

```diff
@@ -0,0 +1,31 @@
+// Reference: https://javascript.plainenglish.io/sending-emails-with-nodemailer-in-next-js-ccada06abfc9
+
+const nodemailer = require("nodemailer");
+//-----------------------------------------------------------------------------
+export async function sendMail(subject, toEmail, otpText) {
+  const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+      user: process.env.NODE_MAILER_EMAIL,
+      pass: process.env.NODE_MAILER_PASSWORD,
```

Comment:
Are these new environment variables? Do they need to be added to production or have they already been added?

Reply:
Confirming per conversation with Joe that these variables have already been added to production.